cdc/capture.go

```diff
@@ -42,6 +44,10 @@ type Capture struct {
 	ownerWorker  *ownerImpl
 
 	processors map[string]*processor
+	procState  struct {
+		sync.Mutex
+		closed int32
```

Comment:
what 's the point to use `atomic**` fro closed in the flowing codes, since it's protected by Mutex

Reply:
It is used to guarantee operations in `OnStopProcessor` not executed if the `Capture` is closed, but since these operations are reentrant, we can remove this